net/mdns-repeater/src/opnsense/service/templates/OPNsense/MDNSRepeater/mdnsrepeater

```diff
@@ -1,6 +1,6 @@
 {% if helpers.exists('OPNsense.MDNSRepeater.enabled') and OPNsense.MDNSRepeater.enabled == '1' %}
 {% from 'OPNsense/Macros/interface.macro' import physical_interface %}
-mdns_repeater_enable="YES"
+mdns_repeater_enable="{% if OPNsense.MDNSRepeater.enablecarp == '1' %}NO{% else %}YES{% endif %}"
```

Comment:
So that’s where onestart comes from. We have to find another way.

Reply:
Yes, I couldn't figure out any other way.

The problem is after a reboot the hook script gets called before the service is auto-started, so it will try to stop the service which isn't running yet. Not sure if there is any way to properly delay the hook script.